src/Build.UnitTests/Definition/ToolsetReader_Tests.cs

```diff
@@ -92,7 +92,7 @@ private void DeleteTestRegistryKey()
         /// Test to make sure machine.config file has the section registered
         /// and we are picking it up from there.
         /// </summary>
-        [Fact]
+        [WindowsOnlyFact]
         public void GetToolsetDataFromConfiguration_SectionNotRegisteredInConfigFile()
         {
             if (!NativeMethodsShared.IsWindows)
```

Comment:
Looks like these conditions could be cleaned up too.

Reply:
Removed obsolete conditions too